code/osapi/DebugWindow.cpp

```diff
@@ -0,0 +1,175 @@
+//
+//
+
+#include "DebugWindow.h"
+
+#include "globalincs/alphacolors.h"
+
+namespace {
+uint32_t get_debug_display() {
+	// If there are two or more monitors then we would like to display the window on the second monitor but it shouldn't
+	// be shown on the same monitor that the game is running on
+	// If there is only then then we'll just display it on that one
+	auto numDisplays = SDL_GetNumVideoDisplays();
+	
+	if (numDisplays < 1) {
+		// Some kind of error
```

Comment:
Well, that code is the error handler. IMO simply defaulting to the primary screen if there was an error is a good solution.